sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -2849,4 +2849,23 @@ private[sql] object QueryExecutionErrors extends QueryErrorsBase {
         "data" -> data,
         "enumString" -> enumString))
   }
+
+  def hllInvalidLogK(function: String, min: String, max: String, value: String): Throwable = {
+    new SparkRuntimeException(
+      errorClass = "HLL_INVALID_LOG_K",
+      messageParameters = Map(
+        "function" -> function,
```

Comment:
Please, quote the function name by `toSQLId`

Reply:
Sounds good, done.